lib/metanorma/cli/command.rb

```diff
@@ -10,7 +10,7 @@ class Command < Thor
       option :type, aliases: "-t", required: true, desc: "Document type"
       option :doctype, aliases: "-d", required: true, desc: "Metanorma doctype"
       option :overwrite, aliases: "-r", desc: "Overwrite existing document"
-      option :template, aliases: "-g", desc: "Git hosted remote template skeleton"
+      option :template, aliases: "-g", desc: "Git hosted remote or local FS template skeleton"
```

Comment:
Should we change this to `-l` for "location" instead?

Reply:
Make sense, done